app/controllers/api/users/responsibilities_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Responsibilities < Api::BaseController
+
+  def create
+    responsibility = Responsibility.new responsibility_params
+    render json: responsibility, ResponsibilityIndexSerializer
+
```

Comment:
Missing an `end` here, and should use the `ResponsibilityBaseSerializer` for now.